twine/settings.py

```diff
@@ -82,6 +82,9 @@ def __init__(self,
         :param str cacert:
             The path to the bundle of certificates used to verify the TLS
             connection to the package index.
+        :param bool trusted:
+            Accept TLS connection with unknown server CA certificates.
+            Don't use with cacert parameter
```

Comment:
Let's update this to say `Prevents the cacert parameter from being used if set to True`

Reply:
Done